sphinx/screens-detail/subscription/subscription/src/main/java/chat/sphinx/subscription/ui/widgets/SphinxRadioGroup.kt

```diff
@@ -0,0 +1,339 @@
+package chat.sphinx.subscription.ui.widgets
+
+import android.content.Context
+import android.text.TextUtils
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import android.view.accessibility.AccessibilityNodeInfo
+import android.view.inputmethod.InputMethodManager
+import android.widget.CompoundButton
+import android.widget.EditText
+import android.widget.RadioButton
+import androidx.annotation.IdRes
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import chat.sphinx.subscription.ui.widgets.SphinxRadioGroup
+
+class SphinxRadioGroup @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr, defStyleRes) {
```

Comment:
Remove Use of `SphinxRadioGroup` custom class.

Custom views such as this:
 - Lock things to the Android platform
 - Does not promote extensibility
 - Needs testing
 - Couples the View and State

This functionality could easily be achieved using the rest of the application's already implemented MVI pattern (`ViewState`) to describe the state of the view, and update it upon reaction to events (such as when a user clicks 1 check box, the rest of them on the view state would change to `checked = false`). Utilizing `ViewState` allows for use of primitive views (`ConstraintLayout`, `CheckBox`, `TextView`) that are already very well tested such that we can de-couple the state from the View by simply rendering things based off of what is described.

Making views and fragments as "dumb" as possible is the primary goal.